cmd/mimir/help-all.txt.tmpl

```diff
@@ -361,6 +361,8 @@ Usage of ./cmd/mimir/mimir:
     	Backend storage to use for the ring. Supported values are: consul, etcd, inmemory, memberlist, multi. (default "memberlist")
   -alertmanager.sharding-ring.zone-awareness-enabled
     	True to enable zone-awareness and replicate alerts across different availability zones.
+  -alertmanager.state-read-timeout duration
+    	[experimental] Timeout for reading the state from object storage during the initial sync. 0 = no timeout. (default 15s)
```

Comment:
Would this make more sense under the `alertmanager_store` block/config?

Reply:
I'm failing to realize how to add this option to the storage section and having the deadline only affect this specific operation (reading state only in the initial sync).

On another note, updating the storage section results in the following args being added after running `make doc`:
- `-alertmanager-storage.state_read_timeout`
- `-common.storage.state_read_timeout`
- `-ruler-storage.state_read_timeout`
- `-blocks-storage.state_read_timeout`